samples/app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "i18n": "react-translate-scripts"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-gettext-plugin": "^2.2.0",
```

Comment:
You don't need this babel plugin in production right? So it should remain as a dev deps because you extract translations only when you are developing!